### PR TITLE
Fixes #163 - Add a job for ML classification of broken site reports using bugbug

### LINKED ISSUE #163: Add a job for ML classification of broken site reports using bugbug
This ETL script will periodically send a batch of reports to[ bugbug http service](https://github.com/mozilla/bugbug) for classification and store results in BQ.

### PR DESCRIPTION
**Pull request by ksy36**
This ETL job sends a batch of reports to [bugbug http service](https://github.com/mozilla/bugbug) for classification (valid/invalid) and stores results in BQ.

Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is
  referenced, the pull request should include the bug number in the title)
- [ ] Scan the PR and verify that no changes (particularly to
  `.circleci/config.yml`) will cause environment variables (particularly
  credentials) to be exposed in test logs
- [ ] Ensure the container image will be using permissions granted to
  [telemetry-airflow](https://github.com/mozilla/telemetry-airflow/)
  responsibly.

### CODE REVIEW

File: jobs/broken-site-report-ml/broken_site_report_ml/main.py [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+    response = requests.post(
+        url, headers={"X-Api-Key": "docker-etl"}, json={"reports": reports_list}
+    )
+
+    response.raise_for_status()
+
+    return response.json()
+
+
+def get_reports_classification(model, reports, retry_count=21, retry_sleep=10):
```
Comment (ksy36) [inline jobs/broken-site-report-ml/broken_site_report_ml/main.py:24]: I've reused the same logic for retry/sleep as in https://github.com/mozilla/bugbot/blob/master/bugbot/bugbug_utils.py

**Review by scholtzan — APPROVED** [review]

**Review by denschub — APPROVED** [review]
Look good to me, thanks for working on this!

**Comment by ksy36** [comment]
Thanks! 

@scholtzan wonder if you could trigger CI, please?

**Comment by ksy36** [comment]
Thank you :)

**Pull request merged**